lib/bandit/headers.ex

```diff
@@ -2,6 +2,8 @@ defmodule Bandit.Headers do
   @moduledoc false
   # Conveniences for dealing with headers
 
+  import Bandit.Ports
+
```

Comment:
The implementation of this is pretty simple; I think this is a rare case of `defguardp`